### ISSUE TITLE
Does not support scaling for HiDpi displays properly

### ISSUE THREAD

**Issue by RitwickVerma** [psifidotos/applet-window-buttons#58]
With plasma 5.17, fractional scaling support was improved a lot. I'm using a HiDpi display with a scaling of 1.2. The problem is, Whenever I logout, the thickness margin stops working for some reason. 
eg. I have thickness margin 8 px. I log out and the margin still shows 8 yet, the window buttons are extremely tiny. To fix I have to change the margin, apply, and change it back to 8, apply again. It sticks after sleep but not after log out.

**Comment by psifidotos** [comment]
1. There is no thickness margin in master version only, only button size. Please retest and confirm
2. Upload screenshots explaining the issue
3. X11 or wayland?

**Comment by RitwickVerma** [comment]
I am running arch so I installed plasma5-applets-window-buttons from the AUR which was having the said issue. 

![Screenshot_20191026_031219](https://user-images.githubusercontent.com/16993844/67618072-efb4ae80-f79e-11e9-9d37-8f575f5aac1a.png)

![Screenshot_20191026_031249](https://user-images.githubusercontent.com/16993844/67618073-efb4ae80-f79e-11e9-8d96-56b17755ca5a.png)

Now, I've installed plasma5-applets-window-buttons-git version which only has button size option as you said and it's working perfectly. The problem is resolved, thanks a lot. For reference, I'm using X11.
